src/Kunstmaan/NodeSearchBundle/Configuration/NodePagesConfiguration.php

```diff
@@ -675,32 +640,17 @@ protected function getUTCDateTime(\DateTime $dateTime)
     /**
      * Removes all HTML markup & decode HTML entities
      *
+     * @deprecated since 5.1. Use "Kunstmaan\NodeSearchBundle\Helper\SearchRenderHelper::removeHtml" instead.
+     *
      * @param $text
      *
      * @return string
      */
     protected function removeHtml($text)
     {
-        if (!trim($text)) {
-            return '';
-        }
+        @trigger_error(sprintf('The "%s" method is deprecated in NodeSearchBundle 5.1 and will be removed in NodeSearchBundle 6.0.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
See comment above about including the possible solution in the deprecation message